### ISSUE TITLE
cpansa_dev.json is broken

### ISSUE THREAD

**Issue by stigtsp** [CPAN-Security/cpansa-feed#1]
```
$ curl https://cpan-security.github.io/cpansa-feed/cpansa_dev.json | jq
jq: parse error: Invalid numeric literal at line 7, column 16
```

**Comment by stigtsp** [comment]
It's referenced by MetaCPAN for instance: https://github.com/metacpan/metacpan-api/pull/1290

**Comment by garu** [comment]
Fixed in 9b1323e.
